.github/workflows/main.yml

```diff
@@ -10,9 +10,22 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 0
+    - name: Init Coveralls
+      shell: bash
+      run: |
+          COVERALLS_TOKEN=${{ secrets.COVERALLS_REPO_TOKEN }}
+          if [[ -z "${COVERALLS_TOKEN}" ]];
+          then
+             echo "Coveralls token not available"
+             COVERALLS_SKIP=true
+          else
+             echo "Coveralls token available"
+             COVERALLS_SKIP=false
+          fi
+          echo ::set-env name=COVERALLS_SKIP::${COVERALLS_SKIP}
```

Comment:
Was a bit unfamiliar with how github handled passing dynamic variables in between build steps in a workflow. Interesting that they went this log syntax: https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
Yes, so was I. I initially tried `set-output` but couldn't get it working so went with `set-env`, it doesn't make much difference in this case. It is an "interesting" syntax indeed, I don't think I'm a fan but maybe I'll come around to it.